client/ayon_resolve/api/lib.py

```diff
@@ -906,6 +908,32 @@ def _convert_resolve_list_type(resolve_list):
     return [resolve_list[i] for i in sorted(resolve_list.keys())]
 
 
+def get_clip_resolution_from_media_pool(timeline_item_data):
+    """Return the clip resolution from media pool data.
+
+    Args:
+        timeline_item_data (dict): Timeline item to investigate.
+
+    Returns:
+        resolution_info (dict): The parsed resolution data.
+    """
+    clip_item = timeline_item_data["clip"]["item"]
+    media_pool_item = clip_item.GetMediaPoolItem()
+    clip_properties = media_pool_item.GetClipProperty()
+
+    try:
+        width, height = clip_properties["Resolution"].split("x")
+    except (KeyError, ValueError):
+        width = height = None
+
+    try:
+        pixel_aspect = int(clip_properties["PAR"]) # Pixel Aspect Resolution
+    except(KeyError, ValueError):
```

Comment:
When would this error?

Reply:
I looked into that a bit more, it's not quite clear what Resolve is doing. It seems to store PAR as a string.
![image](https://github.com/user-attachments/assets/66e54dc4-cab4-4377-9e5a-6f133aba6ccb)

So I've come ahead with this approach:
https://github.com/ynput/ayon-resolve/pull/15/files#diff-634f1d18d81de9d69a1b8517faa8142e794ee030651aee279a5982d806d20269R20
(btw this helped me a lot https://www.unravel.com.au/aspect-ratio-cheat-sheet)

I kept a default to `1.0` in case of a weird behavior from Resolve.